transmart-rest-api/src/test/groovy/org/transmartproject/rest/serialization/tabular/TabularResultSPSSSerializerSpec.groovy

```diff
@@ -42,7 +42,7 @@ class TabularResultSPSSSerializerSpec extends Specification {
         def columns = ImmutableList.copyOf([] as List<DataColumn>)
 
         when: 'producing spss files for empty table'
-        new TabularResultSPSSSerializer(Mock(User), Mock(ZipOutputStream), columns)
+        new TabularResultSPSSSerializer(Mock(User), Mock(ZipOutputStream), columns, 'testFile')
```

Comment:
Please add a check for the file (zip, dir. and file) names to the tests.

Reply:
I added a check like this to the [export e2e tests](https://github.com/thehyve/transmart-core/blob/export-file-naming/transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/DataExportSpec.groovy), because this test is meant to fail before the files are created

Reply:
Looks fine.
Does it satisfy the TMT-213 description? What about timestamp as a filename?